### ISSUE TITLE
[BUG] VA Facility and Status Updates Banner Dismissible function misaligned (Safari) 

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#32630]
## What happened?
_When mobile users are looking at VA.gov homepage, the "Va Facility and Status Updates banner dismissible feature is hidden off to the side on Safari browsers.  Users have to swipe right in order for the dismissible feature to show and function_

## Screenshot:
![IMG_8941](https://user-images.githubusercontent.com/70410912/140827180-d4011bf9-606e-410d-9bb3-7c10d8763fbf.jpeg)

## Specs:
- Device: Mobile Only - iPhone 
- Browser: Safari 

## Steps to Reproduce
- URL: va.gov on a mobile device, swipe right when page loads. 

## Desired behavior
_Describe in detail what was supposed to happen; reference existing Github issue that contains a solution if possible_

## Acceptance Criteria
- [ ] _Dismissible element should be visible for mobile users on the banner.  Users shouldn't have to swipe right in order to see the dismissible feature.

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to Epic** (what body of work is this a part of? possibly `Ongoing Maintenance`)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with `Bug`**

**Comment by tmac9494** [comment]
@brianalloyd I'm having trouble replicating this consistently, do we have information on which ios version/iphone it was? Replicating this in xcode has been challenging.

**Comment by brianalloyd** [comment]
@tmac9494 I'm experiencing it on my iphone 10x max, the "X" is hidden and you have to swipe right in order to see it and dismiss.

**Comment by tmac9494** [comment]
Found steps to reliably reproduce yesterday afternoon, should have a push finished up this afternoon

**Comment by tmac9494** [comment]
The issue was caused by a "box-sizing: inherit "rule set on the va-alert element causing the element to switch to a different box-sizing property than "border-box" which caused the width to not be calculated correctly. I added a "box-sizing: border-box !important" rule on the element and added a max-width of 100vw to prevent the boundary from breaking outside the frame again. Also removed a lot of unnecessary padding between the content and the button.
